comps/asr/whisper/dependency/whisper_server.py

```diff
@@ -106,7 +106,7 @@ async def audio_transcriptions(
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--host", type=str, default="0.0.0.0")
-    parser.add_argument("--port", type=int, default=7066)
+    parser.add_argument("--port", type=int, default=os.getenv("WHISPER_PORT", 7066))
```

Comment:
```suggestion
    parser.add_argument("--port", type=int, default=int(os.getenv("WHISPER_PORT", 7066)))
```

Reply:
Thought about that but `type=int` takes care of that